### ISSUE TITLE
ApplicationLauncherGroup does not forward custom props to DOM

### ISSUE THREAD

**Issue by jschuler** [patternfly/patternfly-react#6454]
ApplicationLauncherGroup does not forward custom props to DOM

Looking at the code, it forwards it to the underlying DropdownGroup component, but then DropdownGroup does not accept other props.

For example, i cannot do
```
<ApplicationLauncherGroup onMouseOver={() => console.log('mouse over)}>Group</ApplicationLauncherGroup>
```

**Comment by boaz0** [comment]
@tlabaj / @mcarrano can you close this issue as #6521 was merged?
Thanks!

**Comment by tlabaj** [comment]
closed by #6521
